frontend/src/components/layout/Header.css

```diff
@@ -1,8 +1,12 @@
+/* Header.css */
```

Comment:
you dont want comments in your production code

Reply:
done!